tiledb/sm/rest/curl.cc

```diff
@@ -241,6 +258,27 @@ Status Curl::init(
   return Status::Ok();
 }
 
+size_t Curl::OnReceiveData(
+    void* res_data, size_t tSize, size_t tCount, void* userdata) {
+  size_t length = tSize * tCount, index = 0;
```

Comment:
declare variables on their own line, for consistency with the rest of the codebase.